spec/schema_registry_spec.rb

```diff
@@ -193,6 +193,23 @@
         end
       end
     end
+
+    context "server error" do
+      let(:compatibility) { 'BOTH' }
+
+      let(:old_json) do
+        { type: 'record', name: 'event', fields: [{ name: 'attribute', type: { type: 'record', name: 'reference', fields: [{ name: 'id', type: 'string' }] } }] }.to_json
```

Comment:
nit: These lines are getting pretty long. Can you split these hash literals across multiple lines?

Reply:
fixed